modules/cli/apply/apply.go

```diff
@@ -69,12 +70,15 @@ func Apply(ctx context.Context, client *client.Getter, name string, baseDirs []s
 	}
 
 	foundBundle := false
+
 	for i, baseDir := range baseDirs {
 		matches, err := globDirs(baseDir)
 		if err != nil {
 			return fmt.Errorf("invalid path glob %s: %w", baseDir, err)
 		}
 		for _, baseDir := range matches {
+			logrus.Infof("*** baseDir %v", baseDir)
```

Comment:
need to remove log lines

Reply:
yes added them for test; would remove them